src/LazyList/index.js

```diff
@@ -15,7 +15,8 @@ export default class LazyList extends React.Component {
     super(props);
 
     this.state = {
-      renderIndices: Range(0, 0)
```

Comment:
The functionality seems OK, but having the it located in `LazyList` would violate this component's concerns. `LazyList` should only care about the loading and unloading of list items based on scroll position. It does not have knowledge of lines, or the contents of its lists, or how to jump to a certain line. This functionality should probably be in the `LogViewer` component, by using the line knowledge to change the scroll position, which would then cause `LazyList` to update its loaded containers, which should already be handled.